src/mz/src/utils.rs

```diff
@@ -88,3 +88,39 @@ pub(crate) fn exit_with_fail_message(message: ExitMessage) -> ! {
     }
     exit(1);
 }
+
+/// Layouts an UUID.
+/// Frontegg API needs the UUID with the layout.
+fn layout_uuid(uuid: String) -> String {
```

Comment:
It feels kinda weird here to assume that the string is always going to have the wholly-correct format. Can this be structured such that it parses the `Uuid` and then formats it as `uuid.as_simple().to_string()`?

I guess ideally, the `FronteggAPIToken` could parse the mzp_ string into the two UUIDs and validate that they parse right; then everything coming out of it is guaranteed to be valid.

Reply:
Added the `uuid` crate to do the parsing and checks.